honcho-cli/src/honcho_cli/commands/setup.py

```
@@ -0,0 +1,253 @@
+
+            client = Honcho(base_url=config.base_url, api_key=config.api_key, workspace_id=config.workspace_id)
+            client.get_configuration()
+            ws_ok = True
+            _add("Workspace reachable", True, config.workspace_id)
+        except Exception as e:
+            _add("Workspace reachable", False, f"{config.workspace_id}: {e}")
+        if ws_ok:
+            try:
+                q = client.queue_status()
+                _add("Queue health", True, f"{q.completed_work_units}/{q.total_work_units} completed, {q.pending_work_units} pending")
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)